src/index.ts

```diff
@@ -1,5 +1,6 @@
 import Checkpoint from './checkpoint';
 export { LogLevel } from './utils/logger';
+export { parseEvent } from './utils/events';
```

Comment:
Any reason to expose this function at root level? Shouldn't this be exposed by StarkNet provider file?